package.json

```diff
@@ -117,6 +120,7 @@
     "react-day-picker": "6.0.2",
     "react-docgen": "2.13.0",
     "react-dom": "15.4.0",
+    "react-hot-loader": "1.3.1",
```

Comment:
> Docker builds are faster 

thats what I had assumed at first as well. Turns out we only run an npm install when `package.json` changes, so this shouldn't affect much.